src/pages/sessionCreator.js

```diff
@@ -0,0 +1,30 @@
+import Stepper from "@/components/Stepper";
+import TestDetails from "@/components/Steps/TestDetails";
+import Timeline from "@/components/Steps/Timeline";
+import StudentDetails from "@/components/Steps/studentDetails";
+
+import React, { useState } from "react";
+
+export default function sessionCreator() {
+  const [activestep, setActiveStep] = useState(0);
+  const activeForm = () => {
+    if (activestep === 0) {
```

Comment:
Use an enum with string names so it's clear what each step is

Reply:
Added enum and used them